compiler/rustc_codegen_llvm/src/attributes.rs

```diff
@@ -335,6 +337,17 @@ pub fn from_fn_attrs(cx: &CodegenCx<'ll, 'tcx>, llfn: &'ll Value, instance: ty::
 }
 
 pub fn provide_both(providers: &mut Providers) {
+    providers.needs_gdb_debug_scripts_section = |tcx, cnum| {
```

Comment:
Please move this to rustc_codegen_ssa. This could also be used for alternative codegen backends.